algosdk/v2client/models/application_params.py

```diff
@@ -193,6 +199,29 @@ def global_state(self, global_state):
 
         self._global_state = global_state
 
+    @property
+    def version(self):
+        """Gets the version of this Application programs.  # noqa: E501
+
+        Represents the number of updates to the application programs.  # noqa: E501
+
+        :return: The version of this Application programs.  # noqa: E501
+        :rtype: int
+        """
+        return self._version
+
+    @version.setter
```

Comment:
Does version need a setter?

Reply:
It gets used in the initializer, I was just following the pattern from other fields, perhaps could avoid?

Reply:
Nah, stay with the pattern.